### PR TITLE
feat: add monitor icon

### PR DESCRIPTION
**Pull request by steue**
- [ ] **BREAKING CHANGE?** 

## Description

Adds the following:
- `26/monitor.svg`

## Checklist

* [x] :ok_hand: SVG updates are Garden Designer approved (add the
  designer as a reviewer)
* [x] :globe_with_meridians: SVG demo is up-to-date (`yarn start`)
* [x] :black_medium_small_square: Renders as expected in "dark" mode
* [x] :white_large_square: Renders as expected @ 2x scale

### CODE REVIEW

**Comment by steue** [comment]
wondering if i should wait till we finish the color palette icon? should be ready early next week.

File: src/26/monitor.svg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="26" height="26" focusable="false" viewBox="0 0 26 26">
+  <g fill="none">
+    <path d="M0 0h26v26H0z"/>
```
Comment (jzempel) [inline src/26/monitor.svg:3]: This path is superfluous and should be removed. Once that is gone, the `<g>` is no longer needed, either.

**Review by jzempel — CHANGES_REQUESTED** [review]

File: src/26/monitor.svg (hunk 1 above)
Comment (steue) [reply src/26/monitor.svg]: great catch!

**Review by jzempel — APPROVED** [review]
I'll sit on this until the color palette icon comes through 👍

**Pull request merged**